### DISCUSSION TITLE
Column grouping makes the cells of the new row take the class of the cells below them

### DISCUSSION THREAD

**Discussion by marcoselopez** [KevinVandy/material-react-table#464] [Q&A]
Hello everyone!

So I'm having this issue and I don't know how to change it.
I have a table that has some grouped columns, where there is a parent column and that parent columng has children sub-columns, exactly like this:

![image](https://user-images.githubusercontent.com/83432730/231267258-25590af4-1c85-4bde-9bc6-3464337bdfaf.png)

The problem with this, is that when the new header row is created, all of the cells which are not defined are taking the className of the cell below them, so it looks like this:

![image](https://user-images.githubusercontent.com/83432730/231267147-beb022cc-7713-4ab5-87bb-6a206343317e.png)

I've tried experimenting with different props and styles but I can't make the new head row to not take any styles unless I specify it directly in the column object, is there a way to achieve that?

**Comment by KevinVandy** [comment]
Interesting, I'll look into this inconsistency

**Reply by marcoselopez** [reply]
Thank you so much Kevin, I'll tried to "separate" the classes for each cell in each row but I was unsuccesfull. If you need any othe info on this matter please tell me.

Have a great day!
